src/device/usbd.c

```diff
@@ -61,6 +61,9 @@ typedef struct {
 
 static usbd_device_t _usbd_dev = { 0 };
 
+// Invalid driver ID in itf2drv[] ep2drv[][] mapping
+enum { DRVID_INVALID = 0xff };

```

Comment:
How about if we also explicitly declare 0xff as unsigned by using 0xFFu? (This is creating implicit type conversion warnings)

Reply:
Yes, it should have u suffix. Thanks